x-pack/plugins/observability_solution/entity_manager/server/routes/entities/get.ts

```diff
@@ -33,7 +33,7 @@ export function getEntityDefinitionRoute<T extends RequestHandlerContext>({
           page: req.query.page ?? 1,
           perPage: req.query.perPage ?? 10,
         });
-        return res.ok({ body: definitions });
+        return res.ok({ body: { definitions } });
```

Comment:
what's the impact of this change?

Reply:
None since this api is not exposed or used at the moment